src/third-party/PhishingController.ts

```diff
@@ -225,15 +222,20 @@ export class PhishingController extends BaseController<
       return;
     }
 
-    this.lastFetched = Date.now();
-
     const configs: EthPhishingDetectConfig[] = [];
 
-    // We ignore network failures here instead of bubbling them up
-    const [metamaskConfigLegacy, phishfortHotlist] = await Promise.all([
-      this.queryConfig<EthPhishingResponse>(METAMASK_CONFIG_URL),
-      this.queryConfig<string[]>(PHISHFORT_HOTLIST_URL),
-    ]);
+    let metamaskConfigLegacy;
+    let phishfortHotlist;
+    try {
+      [metamaskConfigLegacy, phishfortHotlist] = await Promise.all([
+        this.queryConfig<EthPhishingResponse>(METAMASK_CONFIG_URL),
+        this.queryConfig<string[]>(PHISHFORT_HOTLIST_URL),
+      ]);
+    } finally {
```

Comment:
Do we also want to catch errors and log them somewhere so we can be alerted if these APIs go down

Reply:
Note that there is no `catch` block here, so this change does not affect error handling.

There is a separate issue for improving error handling: https://github.com/MetaMask/controllers/issues/923

Reply:
Yeah I meant should we add a catch block. But I see you've already got a ticket for improving error handling separately.